### PR TITLE
Calculator console App by Alain Nshimirimana

### PR DESCRIPTION
**Pull request by AlainNshimirimana**
This is my calculator console app. Please let me know what you think. My discord is alaindemain#6387 if it's needed

### CODE REVIEW

File: CalculatorConsoleApp.AlainNshimirimana/Program.cs [hunk 1]
```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using System.Buffers;
+using System.Collections.Generic;
```
Comment (TheCSharpAcademy) [inline CalculatorConsoleApp.AlainNshimirimana/Program.cs:3]: please remove unused usings

File: CalculatorConsoleApp.AlainNshimirimana/Program.cs [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+void Division()
+{
+    operationType = "Division";
+    Console.ForegroundColor = ConsoleColor.Cyan;
+    Console.WriteLine("DIVISION");
+    Console.ResetColor();
+    Console.WriteLine("--------");
+    Console.Write("First number: ");
+    num1 = double.Parse(Console.ReadLine());
```
Comment (TheCSharpAcademy) [inline CalculatorConsoleApp.AlainNshimirimana/Program.cs:129]: I'm getting this error when trying to input something other than a number. 
image

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

File: CalculatorConsoleApp.AlainNshimirimana/Program.cs (hunk 2 above)
Comment (AlainNshimirimana) [reply CalculatorConsoleApp.AlainNshimirimana/Program.cs]: Fixed or at least made it so the app checks that the user entered a number

**Comment by AlainNshimirimana** [comment]
Made the requested changes.

**Pull request merged**